src/documentation/markdown/2020/lar-formatting.md

```diff
@@ -1,137 +1,165 @@
+### Delete a row
+  - Selecting a record in [`Saved Records`](#saved-records) will populate that row's data in the [`Editing`](#editing-a-record) section of the app.
+  - Click the [`Delete`](#row-actions) button to remove the selected row.
+  - You will be prompted to confirm this action.
 
-If the message “The macro may not be available in this workbook or all macros may be disabled” appears, then the macro settings must be updated.
+### Download your LAR file
+  - Use the [`Download File`](#file-actions) button to download a pipe-delimited, UTF-8 encoded, `.txt` file containing the TS and LAR data shown in the [`Saved Records`](#saved-records) section. Unsaved data entered in the [`Editing`](#editing-a-record) section is not included in the downloaded LAR file.
+  - The file is saved to your browser's default download directory.
+  - The data is now formatted into a pipe-delimited text file ([as shown below](#lar-file-output)) and ready to be submitted to the HMDA Platform.
 
-To enable the HMDA LAR Formatting Tool macro:
+## Editing an existing LAR file
 
-1.	Open the HMDA\_LAR\_Formatting\_Tool workbook.
+### Upload your file
+Use the [`Upload File`](#file-actions) button to select your file. All processing is done in your browser, no data is sent to the CFPB.
 
-2.	Select **Tools**.
+Note: Rows without a valid `Record Identifier` are filtered out during data parsing and presented in a [`Records for review`](#please-review) section. You should review and correct these records.
 
-3.	Select **Options**.
+### Search the TS/LAR data
+Use the [`Saved Records`](#saved-records) area to find records relevant to your focus.
 
-4.	Within the Options dialog box, select the **Security** tab.
+  - You can search the text of your file using the `Search TS` and `Search LAR` boxes.
+  - You can filter which fields are shown using the `Filter columns/Filter by label` boxes.
 
-5.	Select **Macro Security**.
+### Edit a row
+Selecting a record in [`Saved Records`](#saved-records) will populate that row's data in the [`Editing`](#editing-a-record) section of the app.
 
-6.	In the Security dialog box, select the **Medium** option.
- 
- a. This will allow you to choose which macros to run.
+  - [`Parsed Values`](#parsed-values) provides examples of field inputs, field and input descriptions, and easy selectors for enumerated fields.
+  - [`Pipe-Delimited Values`](#pipe-delimited-values) provides the row's raw text, as it will appear in your downloaded LAR file.
+    - Clicking in a field will highlight that value in the [`Parsed Values`](#parsed-values) section for quick editing.
 
-7.	Select **OK** to close the Security dialog box.
+### Save changes
+After making your changes, use the [`Download File`](#file-actions) button to download your LAR file.
 
-8.	Select **OK** to close the Options dialog box.
+The data is now formatted into a pipe-delimited text file ([as shown below](#lar-file-output)) and ready to be submitted to the HMDA Platform.
```

Comment:
`[as shown below](#lar-file-output)` doesn't link to anything.

Reply:
updated